js/transform.js

```diff
@@ -119,8 +119,18 @@ function getNameAndTaxYear(inputDom) {
 function getListOfForms(inputDom) {
     var returnData = inputDom.getElementsByTagName('ReturnData')[0];
     var children = returnData ? Array.prototype.slice.call(returnData.children) : [];
+    var childCounts = {};
     return children.map(function(child) {
-        return child.nodeName;
+        // If the element has not been counted yet, initialize its count
+        if (!Object.hasOwn(childCounts, child.nodeName)) {
+            childCounts[child.nodeName] = 0;
+        } else {
+            // If the element has already been counted, increment its count
+            childCounts[child.nodeName]++;
+        }
+
+        // Append the count to the element to make it unique
+        return `${child.nodeName}_${childCounts[child.nodeName]}`;
```

Comment:
[Dust] I was thinking about your comment below about making this future-proof in case a form has an underscore in its name. How large a refactor is it to return an object of the following schema `{ formName: child.nodeName, formId: childCounts[child.nodeName] }`? This could help make things more robust, but of course changes some assumptions.

If you're not a fan of that, I could see a different character being more robust than an underscore, something that is less likely to be in a tax form. Like an `@` or `~` or `#`